warnsystem/warnsystem.py

```diff
@@ -976,7 +976,7 @@ def _set_log(self, sentry: "Log"):
         """
         Softban the member from the server.
 
-        This means that the user will be banned and immediately unbanned, so it will purge his\
+        This means that the user will be banned and immediately unbanned, so it will purge their\
```

Comment:
"the user" is also singular here.

Reply:
Again, their can be used as plural or singular.